packages/components/src/progress-bar/README.md

```diff
@@ -0,0 +1,47 @@
+# ProgressBar
+
+A simple horizontal progress bar component.
+
+Supports two modes: determinate and indeterminate. A progress bar is determinate when a specific progress value has been specified (from 0 to 100), and indeterminate when a value hasn't been specified.
+
+### Usage
+
+```jsx
+/**
+ * WordPress dependencies
+ */
+import { __experimentalProgressBar as ProgressBar } from '@wordpress/components';
```

Comment:
Hmm, I'm wondering about whether we should really be showing these in the code snippets if it involves an `unlock()` 🤔 It's almost like we're endorsing it.

[`DropdownMenuV2`](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-dropdownmenu-v2--default) is the first of our experimental components involving a lock/unlock, and for that one:

- The component is shown in the Storybook, but without a `import` code snippet.
- The component readme is not shown in the public docs (i.e. not in the manifest.json).

I don't remember having really discussed this with Marco, but maybe we should stick with the same pattern for now? I'll discuss it with the team, but we soon might need to add a new Storybook grouping like "Components (Private)" or something so we can provide `unlock()` instructions for Gutenberg devs, and also avoid confusion for third-party devs.

Reply:
Good point. I've removed the example that contains unlocking, removed the README from the public docs, and verified that the import is not present in Storybook.